File: backend-go/internal/modules/system/entity/sys_print_template.go
package entity

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type SysPrintTemplate struct {
	ID           common.Int64String `gorm:"column:id;primaryKey" json:"id"`
	Code         string             `gorm:"column:code" json:"code"`
	Name         string             `gorm:"column:name" json:"name"`
	PaperType    string             `gorm:"column:paper_type" json:"paperType"`
	TemplateJSON string             `gorm:"column:template_json" json:"templateJson"`
	Status       int                `gorm:"column:status" json:"status"`
	Deleted      int                `gorm:"column:deleted" json:"deleted"`
	CreatedAt    *time.Time         `gorm:"column:created_at" json:"createdAt"`
	UpdatedAt    *time.Time         `gorm:"column:updated_at" json:"updatedAt"`
}

func (SysPrintTemplate) TableName() string { return "sys_print_template" }

File: backend-python/app/modules/system/entity/sys_print_template.py
from sqlalchemy import BigInteger, Integer, String, Text
from sqlalchemy.orm import Mapped, mapped_column

from app.modules.system.entity.base import Base, TimestampMixin


class SysPrintTemplate(TimestampMixin, Base):
    __tablename__ = "sys_print_template"

    id: Mapped[int] = mapped_column(BigInteger, primary_key=True)
    code: Mapped[str] = mapped_column(String(64))
    name: Mapped[str] = mapped_column(String(100))
    paper_type: Mapped[str] = mapped_column(String(32))
    template_json: Mapped[str] = mapped_column(Text)
    status: Mapped[int] = mapped_column(Integer)
    deleted: Mapped[int] = mapped_column(Integer)
